client/package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:watch": "npx onchange '**/*.{js,ts,jsx,tsx}' -- npm run lint"
   },
```

Comment:
**_Review comment 2_**

Nice start @Akpjunior94 there are some issues:

- The `lint:watch` doesn't work as expected. I should be able to see the lint errors in the terminal immediately after I run the command also, not only when I save a file.
- It seems you are using the lint from nextjs not the ones you supplied in the eslint config.
- I also don't see any wiggly lines in the UI of the code editor for things eslint screams about.

Reply:
@Ifycode will look why its not showing up when we run. wiggly line shows for example when we declare a variable and it is not being used it shows the line and throw an error on the terminal. Will also double check and see why.

Reply:
Ok. Thank you. Although it's not only variables we want scream for. Help look into these too:
- We don't want use of double quotes, we want single quotes in js/ts code and imports.
- We want to semicolon at the end of imports and code, objects etc.

Reply:
`lint:watch` fixed to work as expected. wiggly lines shows up now for double quotes and semicolons

Reply:
Agba 🙌🏽